libtestutils/BlockChainLoader.cpp

```diff
@@ -0,0 +1,101 @@
+/*
+	This file is part of cpp-ethereum.
+
+	cpp-ethereum is free software: you can redistribute it and/or modify
+	it under the terms of the GNU General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	cpp-ethereum is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+	GNU General Public License for more details.
+
+	You should have received a copy of the GNU General Public License
+	along with cpp-ethereum.  If not, see <http://www.gnu.org/licenses/>.
+ */
+/** @file BlockChainLoader.cpp
+ * @author Marek Kotewicz <marek@ethdev.com>
+ * @date 2015
+ */
+
+#include "BlockChainLoader.h"
+#include "StateLoader.h"
+#include "Common.h"
+
+using namespace std;
+using namespace dev;
+using namespace dev::test;
+using namespace dev::eth;
+
+namespace dev
+{
+namespace test
+{
+dev::eth::BlockInfo toBlockInfo(Json::Value const& _json);
+bytes toGenesisBlock(Json::Value const& _json);
+
+}
+}
+
+dev::eth::BlockInfo dev::test::toBlockInfo(Json::Value const& _json)
```

Comment:
> that is a duplication of eth::BlockInfo constructBlock(json_spirit::mObject& _o) in TestHelper.cpp. 

yes, I know, but `constructBlock` is a part of `testeth` executable, and I needed this method in the library.

> The only difference is that I use json_spirit and you use Json lib. May be you should consider to switch to json_spirit so we only use one lib for json files in the client, and can reuse our helper functions.

Changing the code to use json_spirit instead Json lib is quite easy. I can do this, if this is needed. Although Json lib is also used in many other ethereum libraries (eg. web3jsonrpc or solidity)